tests/rptest/tests/pandaproxy_test.py

```diff
@@ -40,6 +40,11 @@ def create_topic_names(count):
     "Content-Type": "application/vnd.kafka.binary.v2+json"
 }
 
+HTTP_CREATE_CONSUMER_HEADERS = {
+    "Accept": "application/vnd.kafka.v2+json",
```

Comment:
NICE! i can finally use curl without base64 encoding.

Reply:
> NICE! i can finally use curl without base64 encoding.

I don't think this does what you think it does. This is a fix, since the format of the body for this request is `json.v2`, but the body of this request still sets up the consumer for `binary.v2`. I don't think support for `json.v2` record payloads is too hard, though.

Reply:
ah got it.